subsys/bluetooth/audio/has.c

```diff
@@ -39,6 +37,8 @@ LOG_MODULE_REGISTER(bt_has, CONFIG_BT_HAS_LOG_LEVEL);
 	!BITS_CHANGED(_new_value, ((has.features & BT_HAS_FEAT_PRESET_SYNC_SUPP) != 0 ? 1 : 0))
 #define FEATURE_IND_PRESETS_UNCHANGED(_new_value) \
 	!BITS_CHANGED(_new_value, ((has.features & BT_HAS_FEAT_INDEPENDENT_PRESETS) != 0 ? 1 : 0))
+#define NOTIFY_AFTER_REBOOT \
+	(BIT(NOTIFY_ACTIVE_INDEX) | BIT(NOTIFY_PRESET_LIST) | BIT(NOTIFY_FEATURES))
```

Comment:
I guess this is a mask, right? Suggest to add `_MASK` suffix, as the current name suggests either a verb or a boolean.